Content.Server/Chat/Systems/ChatSystem.cs

```diff
@@ -228,7 +228,7 @@ private void OnGameChange(GameRunLevelChangedEvent ev)
         _chatManager.ChatMessageToAll(ChatChannel.Radio, message, wrappedMessage, default, false, true, colorOverride);
         if (playSound)
         {
-            SoundSystem.Play(announcementSound?.GetSound() ?? DefaultAnnouncementSound, Filter.Broadcast(), AudioParams.Default.WithVolume(-2f));
+            SoundSystem.Play(announcementSound?.GetSound() ?? DefaultAnnouncementSound, Filter.Broadcast(), announcementSound?.Params ?? AudioParams.Default.WithVolume(-2f));
```

Comment:
This should really use AudioSystem since SoundSystem is deprecated. It would simplify this code a lot also.
You could have `announcementSound` be the actual announcement sound instead of doing this `??` nonsense, and get rid of the `bool playSound` argument and if statement and just have null mean no sound.

Reply:
`bool playSound` used here to control of playing default sound